documentcloud/entities/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+from documentcloud.core.fields import AutoCreatedField, AutoLastModifiedField
+from wikidata.client import Client
+from django.utils.translation import gettext_lazy as _
+from .choices import EntityAccess
+
+
+class Entity(models.Model):
+    wd_entity = None
```

Comment:
It will probably be fine as an instance field, but it will save some memory if every Entity instance shares a single Wikidata client, which contains nothing specific to any particular instance. Let me know what you think.